src/features/offer/components/OfferBody/OfferBody.native.test.tsx

```diff
@@ -45,17 +49,32 @@ const mockCheckInstalledApps = jest.spyOn(InstalledAppsCheck, 'checkInstalledApp
 const mockShareSingle = jest.spyOn(Share, 'shareSingle')
 const mockNativeShare = jest.spyOn(NativeShare, 'share')
 
+const mockUseOffer = jest.fn().mockReturnValue({ data: mockOffer })
+jest.mock('features/offer/api/useOffer', () => ({
+  useOffer: () => mockUseOffer(),
+}))
+
+const mockUseAuthContext = jest.fn().mockReturnValue({ isLoggedIn: true, user: beneficiaryUser })
+jest.mock('features/auth/context/AuthContext', () => ({
+  useAuthContext: () => mockUseAuthContext(),
+}))
+
+const mockUseNetInfo = jest.fn().mockReturnValue({ isConnected: true, isInternetReachable: true })
+jest.mock('libs/network/useNetInfo', () => ({
+  useNetInfo: () => mockUseNetInfo(),
+}))
+
+const onScroll = jest.fn()
+
+const offerId = 1
+
 describe('<OfferBody />', () => {
   beforeAll(() => {
     mockdate.set(new Date(2021, 0, 1))
```

Comment:
l'API de `Date` étant ~~dégueu~~ _améliorable_, je recommande de lui passer une string, qui permet d'avoir un format plus logique

```suggestion
    mockdate.set(new Date("2021-01-01"))
```